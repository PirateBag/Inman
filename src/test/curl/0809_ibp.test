Empty Test File
=== JSON ===
{ "idToSearchFor" : "5" }