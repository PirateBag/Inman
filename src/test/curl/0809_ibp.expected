{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0005: W-005,Axle     0.00 MAN   2   4     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 7 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   1    5        0     6.00     0.00  2025-0901  2025-0913 OPEN MOHEAD NONE     6.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  20    5        0   494.00     0.00  2025-0909  2025-0913 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  16    5       15  -500.00     0.00  2025-0913  2025-0914 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   4    5        0   500.00     0.00  2025-0911  2025-0915 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   6    5        0  1000.00     0.00  2025-1011  2025-1015 OPEN MOHEAD NONE  1500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   8    5        0  1500.00     0.00  2025-1111  2025-1115 OPEN MOHEAD NONE  3000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  10    5        0    50.00     0.00  2025-1211  2025-1215 OPEN MOHEAD NONE  3050.00"
  } ]
}